wordpress-mobile/WordPress-iOS
# Modified remaining services to use WordPressKit locale handling

Supports the resolution of [this issue](https://github.com/wordpress-mobile/WordPressKit-iOS/issues/86) in `WordPressKit`.

I've designated this a Draft PR, unless or until the associated updates to [`WordPressAuthenticator`](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/57) & [`WordPressKit`](https://github.com/wordpress-mobile/WordPressKit-iOS/pull/103) have been properly versioned.

To test:

- Branch should build.
- Tests should pass.
- Existing endpoints should work.
- Modified endpoints should also work without regression.

Update release notes:
